.env.example

```diff
@@ -1,3 +1,3 @@
-BOT_USERNAME=
-BOT_PASSWORD=
-BOT_PREFIX=
+BOT_USERNAME=<facebook email/phone number/username>
+BOT_PASSWORD=<facebook password>
+BOT_PREFIX=<message identifier such as "@sadbot">
```

Comment:
Wait another point. We will probably want to be able to do stuff like "@dailyspace" and stuff too. So there's going to be more than just two prefixes.